ang/civicase/case/actions/services/webforms-case-action.service.js

```diff
@@ -20,9 +21,25 @@
      *
      * @returns {boolean} - true if action is allowed, false otherwise.
      */
-    this.isActionAllowed = function (action, cases, attributes) {
+    function isActionAllowed (action, cases, attributes) {
+      checkIfWebformsExist(action.items, cases);
       // Allow this action on Case details page only.
-      return attributes && attributes.mode === 'case-details';
-    };
+
+      return attributes && attributes.mode === 'case-details' &&
+       checkIfWebformsExist(action.items, cases[0].case_type_id);
+    }
+
+    /**
+     * Checks if any webforms are available for the sent case type id
+     *
+     * @param {Array} webforms list of webforms
+     * @param {string} caseTypeID case type id
+     * @returns {boolean} if any webforms are available for the sent case type id
+     */
+    function checkIfWebformsExist (webforms, caseTypeID) {
+      return !!_.find(webforms, function (webform) {
```

Comment:
[_. includes](https://lodash.com/docs/3.10.1#includes) does exactly this and there's no need to prepend `!!`.

Reply:
I tried, but `_.includes` does not support functions for the matcher, so it cannot be used.